lib/config/db/migrations_performer.js

```diff
@@ -0,0 +1,50 @@
+import path from 'path';
+import fs from 'fs';
+import dotenv from 'dotenv';
+import { Migration } from 'mongration';
+import Q from 'q';
+import config from '../config';
+
+dotenv.config();
+
+const migrationConfig = {
+  hosts: config.get('MONGODB_HOST'),
+  db: config.get('MONGODB_DATABASE'),
+  user: config.get('MONGODB_USER'),
+  password: config.get('MONGODB_PASSWORD'),
+  migrationCollection: 'migrationversion',
```

Comment:
That's a valid point @asiakaczmar. I put the collection name in config. In the nearest future I'm going to fork `mongration` repo and extend its API with possibility to rollback a particular migration. It can be used by `post-checkout` git hook which automatically migrates db to state required by checked-out branch.